addons/web/static/src/core/colorlist/colorlist.scss

```diff
@@ -1,5 +1,58 @@
+@mixin o-colorlist($-entry, $-child) {
+    grid-template-columns: repeat(var(--ColorListField-columns, 6), 1fr);
+    gap: var(--ColorListField-gap, #{map-get($spacers, 2)});
+    width: var(--ColorListField-width, none);
+    padding: var(--ColorListField-padding, 0);
+    box-sizing: content-box;
+    margin-bottom: var(--ColorListField-marginBottom, 0);
+
+    #{$-entry} {
+        #{$-child} {
+            display: block;
+            position: relative;
+            min-width: var(--ColorListField-Entry-minWidth, o-to-rem(20px));
+            border-radius: var(--ColorListField-Entry-borderRadius, #{$border-radius});
+            overflow: hidden;
+            aspect-ratio: var(--ColorListField-Entry-aspectRatio, 1);
+            border-radius: var(--ColorListField-Entry-borderRadius, 100%);
+
+            font-size: var(--ColorListField-Entry-fontSize, smaller);
+
+            &::after {
+                @include o-position-absolute(0, 0 , 0, 0);
+                color: white;
```

Comment:
Be aware of the not-colored badge
<img width="434" height="192" alt="image" src="https://github.com/user-attachments/assets/8cc3c031-fb73-497a-bcb9-73d106bba476" />
... and followup for DM, now close to the merge https://github.com/odoo/odoo/pull/227123